src/actions/shortcuts.ts

```diff
@@ -80,8 +80,23 @@ const shortcutMap: Record<ShortcutName, string[]> = {
   toggleLock: [getShortcutKey("CtrlOrCmd+Shift+L")],
 };
 
-export const getShortcutFromShortcutName = (name: ShortcutName) => {
-  const shortcuts = shortcutMap[name];
+export type CustomShortcutName = string;
+
+let customShortcutMap: Record<CustomShortcutName, string[]> = {};
+
+export const registerCustomShortcuts = (
+  shortcuts: Record<CustomShortcutName, string[]>,
+) => {
+  customShortcutMap = { ...customShortcutMap, ...shortcuts };
+};
```

Comment:
we should add shortcut labels into the Action object itself — for now. Ultimately we'll switch to shortcut parsing so we can specify both the label and the actual shortcut as a string (thus deduplicate the label).

Reply:
I have removed this for a separate PR.